### DISCUSSION TITLE
i have prb wiht dangomodel translations

### DISCUSSION THREAD

**Discussion by Dasturchi-Yigit** [community/community#64348] [Programming Help]
### Body

i create a app name order in it a model name order but when i tried to save some data in that model from the user view it doesn't save it then i tried to save it doesn't even got save from there i don't understand what causing the problem

    class BuyOrder(models.Model):
        user = models.ForeignKey(User, on_delete=models.CASCADE,)
        address = models.ForeignKey(Address, default= True, on_delete=models.CASCADE )
        status = models.IntegerField(choices = status_choices, default=1)
        method = models.CharField(max_length=50, blank=False,)
        total_price = models.FloatField(blank=False, default=0)
        payment_status = models.IntegerField(choices = payment_status_choices, default=3)
        order_id = models.CharField(unique=True, max_length=200, null=True, default=None) 
        datetime_of_payment = models.DateTimeField(default=timezone.now)
        # related to razorpay
        razorpay_order_id = models.CharField(max_length=1000, null=True, blank=True)
        razorpay_payment_id = models.CharField(max_length=1000, null=True, blank=True)
        razorpay_signature = models.CharField(max_length=1000, null=True, blank=True)
    
        def save(self, *args, **kwargs):
            if self.order_id is None and self.datetime_of_payment and self.id:
                self.order_id = self.datetime_of_payment.strftime('COOLBUYORDER%Y%m%dODR') + str(self.id)
                return super().save(*args, **kwargs)
            else:
                pass
    
        def __str__(self):
            return self.user.username + " " + str(self.order_id) + " " + str(self.id)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ALISHERpy** [comment]
yeah,i had it before.just delete .po files and .mo 
refresh console 
here we go. this will work. try it

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
